src/actions/index.js

```diff
@@ -0,0 +1,7 @@
+export const CHANGE_LOOK = 'CHANGE_LOOK';
```

Comment:
I see no point in declaring constants outside action creators so I just put the one you're using right here.